UnoCheck/Checkups/HyperVCheckup.cs

```diff
@@ -0,0 +1,73 @@
+﻿#nullable enable
+
+using DotNetCheck.Models;
+using DotNetCheck.Solutions;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Checkups
+{
+	public class HyperVCheckup : Checkup
+	{
+		public override string Id => "windowshyperv";
+
+		public override string Title => "Windows HyperV Checkup";
+
+		public override bool IsPlatformSupported(Platform platform) => platform == Platform.Windows;
+
+		public override async Task<DiagnosticResult> Examine(SharedState history)
+		{
+			
+			if (!await this.GetHyperVStatusAsync())
+			{
+				return await Task.FromResult(new DiagnosticResult(
+				Status.Error,
+				this,
+				new Suggestion("Activate HyperV on this computer? The computer must be BIOS virtualization enabled first!",
+				new HyperVActivationSolution())));
+			}
+			else
+			{
+				var r = ShellProcessRunner.Run("shutdown", "/a");
+
+				ReportStatus(r.GetOutput(), Status.Error);
+
+
+				ReportStatus($"Hyper-V is configured!", Status.Ok);
```

Comment:
Let's just report the need for the computer to be restarted, or point to the documentation.